### PR TITLE
SAK-48189 - Allow creation of two SakaiPlus Tenants with the same Issuer and no Client ID

### PR DESCRIPTION
**Pull request by csev**

### CODE REVIEW

**Comment by csev** [comment]
There is a TODO - I don't know the right way to get the current Local to call getMessage()

**Comment by csev** [comment]
Hey @ern - I need to know hot to get the Locale of the current logged in user in MainController - there are two TODOs I have no idea how to fix.   And the AccessLint complaint - I have no idea what it wants.

File: plus/tool/src/main/webapp/WEB-INF/templates/index.html [hunk 1]
```diff
@@ -13,6 +13,14 @@ <h1 th:text="#{plus.tool.title.tenants}">About SakaiPlus</h1>
         (<a href="https://github.com/sakaiproject/sakai/blob/master/plus/README.md" target="_blank" th:text="#{plus.tool.documentation.url}">SakaiPlus Documentation</a>)
         </p>
         <p><a class="btn btn-primary" th:href="@{/create}" th:text="#{plus.tool.addtenant}">Add Tenant</a></p>
+        <div th:if="${flashError}" class="alert alert-danger alert-dismissible fade show">
+            <p th:text="${flashError}" >Placeholder</p>
+            <button type="button" class="btn-close" data-bs-dismiss="alert"></button>
```
[the remark this answers is not in the document]
Comment (ottenhoff) [reply plus/tool/src/main/webapp/WEB-INF/templates/index.html:18]: the complaint is that this is just a button with no text indication .... is it just a visual icon supplied by class="btn-close"?

Fix by adding an aria-label="Close error"

**Comment by ottenhoff** [comment]
ResourceLoader.getLocale() ?

**Comment by csev** [comment]
I have figured out the locale thing.  Now to the aria bit.   Grrr.

File: plus/tool/src/main/webapp/WEB-INF/templates/index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (csev) [reply plus/tool/src/main/webapp/WEB-INF/templates/index.html:18]: Sam - Does this need to be translated?

[the remark this answers is not in the document]
Comment (csev) [reply plus/tool/src/main/webapp/WEB-INF/templates/index.html:18]: And yes this is an "X" that comes from Bootstrap.  You would think bootstrap would do this

**Pull request merged**